mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -2407,4 +2407,14 @@
     <string name="debug_drawer_tab_tools_tab_count_private">Private</string>
     <!-- The total tab count category in the tab count section in Tab Tools. -->
     <string name="debug_drawer_tab_tools_tab_count_total">Total</string>
+    <!-- The title of the tab creation tool section in Tab Tools. -->
```

Comment:
Is it just me or this looks like a tool that is targeting developers, and shouldn't have localizable strings? Part of the strings also landed in the [previous batch](https://github.com/mozilla-l10n/android-l10n/pull/720/files#diff-b8d7151f11faa90ad8cfbb96437f96d05602954aa79c1145cb3232f6a5eb6d38R2397).

Reply:
I think that's correct, I'll get an issue on file to double check